entities/build.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -e
+
+# Absolute path to the directory containing this script
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+VERBOSE=false
+QUIET=false
+export VERBOSE
+export QUIET
+
+HTML_TEMP=${HTML_TEMP:-$DIR/.temp}
+export HTML_TEMP
+
+for arg in "$@"
+do
+  case $arg in
+    -h|--help)
+      echo "Usage: $0 [-h|--help]"
+      echo "Usage: $0 [-n|--no-update] [-q|--quiet] [-v|--verbose]"
+      echo
+      echo "  -h|--help       Show this usage statement."
+      echo "  -q|--quiet      Don't emit any messages except errors/warnings."
+      echo "  -v|--verbose    Show verbose output from every build step."
+      exit 0
+      ;;
+    -q|--quiet)
+      QUIET=true
+      VERBOSE=false
+      ;;
+    -v|--verbose)
+      VERBOSE=true
+      QUIET=false
+      set -vx
+      ;;
+    *)
+      ;;
+  esac
+done
+
+rm -rf $HTML_TEMP && mkdir -p $HTML_TEMP
+rm -rf entities-dtd.url entities.inc entities.json
+
+# Fetch unicode.xml
+$QUIET || echo "Downloading unicode.xml (can take a short time, depending on your bandwidth)...";
+curl --location $($VERBOSE && echo "-v") $($QUIET && echo "-s") \
+  https://www.w3.org/2003/entities/2007xml/unicode.xml.zip \
+  $( [ -f $HTML_TEMP/unicode.xml ] && echo "--time-cond $HTML_TEMP/unicode.xml" ) \
+  --output $HTML_TEMP/unicode.xml.zip
+[ -f $HTML_TEMP/unicode.xml.zip ] && unzip $($VERBOSE && echo "-v" || echo "-qq") \
+  -o $HTML_TEMP/unicode.xml.zip -d $HTML_TEMP && touch $HTML_TEMP/unicode.xml
+
+# Generate entity files
+$QUIET || echo;
+$QUIET || echo "Generate entities (this always takes a while)...";
+python build-processor.py > $HTML_TEMP/new-entities-unicode.inc;
+[ -s $HTML_TEMP/new-entities-unicode.inc ] && mv -f $HTML_TEMP/new-entities-unicode.inc $HTML_TEMP/entities-unicode.inc; # otherwise, probably http error, just do it again next time
+python build-processor-json.py > $HTML_TEMP/new-entities-unicode-json.inc;
+[ -s $HTML_TEMP/new-entities-unicode-json.inc ] && mv -f $HTML_TEMP/new-entities-unicode-json.inc $HTML_TEMP/json-entities-unicode.inc; # otherwise, probably http error, just do it again next time
+echo '<tbody>' > $HTML_TEMP/entities.inc
```

Comment:
This puts the tbody in a file of its own, and that shows up in the diff of the intermediate source-whatwg-complete that I compared with and without this change. Oddly enough the eventual output is the same, I guess wattsi adds or remove tbody elements for consistency.